/**:
    ros__parameters:
        en_test: true
        en_instaRes: false # 是否启用兑换复活
        en_gimbal_spin: true   # 比赛期间始终默认为true
        en_chassis_spin: false  
        en_chase_enemy: false
        tracking_scope: 5.0
        gimbal_angular_vel: 2.2
        chassis_angular_vel: -6
        chassis_angular_vel_slow: 3
        plan: "B" # A: 启动区附近两点巡逻 B: 抓对面英雄 C: 自家前哨站附近巡逻 
        home_pose:  # 原地
            x: -0.4
            y: 0.86
            yaw: 0.2
        supply_zone_pose:
            x: -3.8
            y: -6.0
            yaw: 0.0
        goal_pose: # 自家前哨站
            x: 7.00539
            y: -4.45441
            yaw: 0.0
        attack_enemy_outpost_pose: #飞坡增益点，击打敌方前哨站
            x: 4.74
            y: 5.54
            yaw: -0.65
        circular_highland_pose: # 环高
            x: 0.0
            y: 0.0
            yaw: 0.0
        # for home_patrol_points
        patrol_1:
            x: 0.76
            y: 1.43
            yaw: -1.34
        patrol_2:
            x: 3.31
            y: 1.27
            yaw: 1.62
        # for hero_patrol_points
        patrol_3:
            x: 3.31
            y: 1.27
            yaw: 1.62
        patrol_4:
            x: 2.10
            y: 1.31
            yaw: 0.5
        # for our_outpost_patrol_points
        patrol_5:
            x: 7.37896
            y: -5.00
            yaw: 1.41501
        patrol_6:
            x: 5.88309
            y: -4.8493
            yaw: -0.422